ova/assets/steps.sh

```diff
@@ -0,0 +1,181 @@
+    CURRENT_W=$(less ${INSTALLER} | grep "WAZUH_VER=")
+    CURRENT_O=$(less ${INSTALLER} | grep "OD_VER=")
+    CURRENT_E=$(less ${INSTALLER} | grep "ELK_VER=")
+
+    # Change wazuh and documentation repository branch
+    sed -i "s/uh\/[0-9]\+\.[0-9]\+\/ex/uh\/${BRANCH}\/ex/g" ${INSTALLER}
+    sed -i "s/on\/[0-9]\+\.[0-9]\+\/re/on\/${BRANCHDOC}\/re/g" ${INSTALLER}
+
+    # Change versions
+    sed -i "s/${CURRENT_W}/WAZUH_VER=\"${WAZUH_VERSION}\"/g" ${INSTALLER}
+    sed -i "s/${CURRENT_O}/OD_VER=\"${OPENDISTRO_VERSION}\"/g" ${INSTALLER}
+    sed -i "s/${CURRENT_E}/ELK_VER=\"${ELK_VERSION}\"/g" ${INSTALLER}
+
+    # Change repository if dev is specified
+    if [ "${PACKAGES_REPOSITORY}" = "dev" ]; then
+        sed -i "s/\[wazuh\]/\[wazuh_pre_release\]/g" ${INSTALLER}
+        sed -i "s/ngpgkey\=https\:\/\/packages\.wazuh\.com/ngpgkey\=https\:\/\/packages\-dev\.wazuh\.com/g" ${INSTALLER}
+        sed -i "s/baseurl\=https\:\/\/packages\.wazuh\.com\/4\.x/baseurl\=https\:\/\/packages\-dev\.wazuh\.com\/pre\-release/g" ${INSTALLER}
+        sed -i "s/https\:\/\/packages\.wazuh\.com\/4\.x\/ui\/kibana/https\:\/\/packages\-dev\.wazuh\.com\/pre\-release\/ui\/kibana/g" ${INSTALLER}
+        sed -i "s/wazuh_kibana-[0-9\.]\+_[0-9\.]\+/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}/g" ${INSTALLER}
+    fi
+
+    # Add Kibana custom user wazuh
+    PATTERN="eval \"rm \/etc\/elasticsearch\/e"
+    HASH="\\\$2y\\\$12\\\$qCvlv3y4\\\.i8nX6wUZOepROVhTWI36H8nH2gxwShHcpIzf0yV1J30K"  # password: wazuh
+    FILE_PATH="\/usr\/share\/elasticsearch\/plugins\/opendistro_security\/securityconfig"
+    sed -i "s/${PATTERN}/sed -i \'\/admin:\/ {N; s\/admin.*\/wazuh:\\\n  hash: \"${HASH}\"\/g}\' ${FILE_PATH}\/internal_users\.yml\n        ${PATTERN}/g" ${INSTALLER}
+    
+    # Set backend_role for wazuh user with all_access
+    sed -i "s/${PATTERN}/sed -i \'s\/\"admin\"\/\"wazuh\"\/g\' ${FILE_PATH}\/roles_mapping\.yml\n        ${PATTERN}/g" ${INSTALLER}
+    sed -i "s/${PATTERN}/sed -i \'s\/\"admin\"\/\"wazuh\"\/g\' ${FILE_PATH}\/internal_users\.yml\n        ${PATTERN}/g" ${INSTALLER}
+
+    # Change user:password in curls
+    sed -i "s/admin:admin/wazuh:wazuh/g" ${INSTALLER}
+
+    # Change user:password in filebeat.yml
+    PATTERN="eval \"curl -so \/etc\/filebeat\/wazuh-template"
+    sed -i "s/${PATTERN}/sed -i \"s\/admin\/wazuh\/g\" \/etc\/filebeat\/filebeat\.yml\n        ${PATTERN}/g" ${INSTALLER}
+
+}
+
+# Edit wazuh installation
+postInstall() {
+
+    # Disable all host modules
+    #sed -i "s/<disabled>no/<disabled>yes/g" /var/ossec/etc/ossec.conf
+    #sed -i "s/<enabled>yes/<enabled>no/g" /var/ossec/etc/ossec.conf
+
+    # Disable /var/log/ files and commands monitoring
+    #sed -i "s/<localfile>/<!--localfile>/g" /var/ossec/etc/ossec.conf
+    #sed -i "s/<\/localfile>/<\/localfile-->/g" /var/ossec/etc/ossec.conf
+
+    # Enable ossec_auth modules
+    #sed -i '/<auth>/ {N; s/<auth>.*yes/<auth>\n\ \ \ \ <disabled>no/g}' /var/ossec/etc/ossec.conf
```

Comment:
This part was done as part of iteration 2 before being moved to iteration 4 waiting to know if it has to be done or not. As it is not necessary for this iteration, I will eliminate it and in the future it will be added if necessary

Fixed in: https://github.com/wazuh/wazuh-packages/pull/763/commits/919d57776b77bdab6e13d4cade8fdc6004bc832c